### PR TITLE
Add turkish translations for encouragement and praise

### PR DESCRIPTION
**Pull request by hyigit2**
We added the Turkish language support for encouragement and praise.

PR task list:
- [x] Update NEWS
- [ ] Add tests (if possible)
- [x] Update documentation with `devtools::document()`

### CODE REVIEW

File: data-raw/i18n_random-phrases.yml [hunk 1]
```diff
@@ -77,6 +77,44 @@ praise:
     - ¡Lo superaste!
     - ¡Excelente respuesta!
     - Deberias sentir mucho orgullo.
+   tr:
```
Comment (gadenbuie) [inline data-raw/i18n_random-phrases.yml:80]:
```suggestion
  tr:
```

Comment (gadenbuie) [reply data-raw/i18n_random-phrases.yml:80]: YAML can be very fussy, I think this one extra space is creating problems.

Comment (hyigit2) [reply data-raw/i18n_random-phrases.yml]: Thank you! I applied the change.

**Comment by coatless** [comment]
Thanks @gadenbuie!

**Comment by gadenbuie** [comment]
🙏 Thank you @hyigit2 and @coatless!

**Pull request merged**